run-integrated-tiger.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -ex
+
+# cd "$(dirname "$0")/.."
+
+log() {
+  local now;
+  now=$(date '+%Y-%m-%d %H:%M:%S')
+  echo "$now $*" 1>&2
+}
+
+# docker network create aio-net
+
+docker run --name gooddata-cn-ce-manual -e APP_LOGLEVEL=INFO -e LICENSE_AND_PRIVACY_POLICY_ACCEPTED=YES \
+    -e BUNDLE_TYPE=gdc -p 3000:3000 -p 5432:5432 --network aio-net -d gooddata/gooddata-cn-ce:2.2.0
+sleep 120
+# HOST=$(docker inspect "gooddata-cn-ce-manual" | jq -r '.[0].NetworkSettings.IPAddress')
+
+export DB_HOST=gooddata-cn-ce-manual \
+export DB_PORT=5432 \
+export DB_USER=demouser \
+export DB_NAME=demo \
+export DB_PASSWORDS="{'pg_local': 'demopass'}"
+docker run -it --rm -e DB_HOST -e DB_PORT -e DB_USER -e DB_NAME -e DB_PASSWORDS --entrypoint '' --network aio-net \
+441851189095.dkr.ecr.eu-central-1.amazonaws.com/nas/data-loader:master /bin/bash -c "/data_load/data_load.py -w goodsales -d pg_local --debug  --skip-download --no-schema-versioning -f"
+
+export HOST=http://localhost:3000
+
+curl $HOST/api/v1/entities/dataSources \
+  -H "Content-Type: application/vnd.gooddata.api+json" \
+  -H "Accept: application/vnd.gooddata.api+json" \
+  -H "Authorization: Bearer YWRtaW46Ym9vdHN0cmFwOmFkbWluMTIz" \
```

Comment:
Please set `YWRtaW46Ym9vdHN0cmFwOmFkbWluMTIz` as variable in the top of this script, as it's called at least twice